metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -175,6 +175,7 @@ final class BloopServers(
     val socket = BloopServers.newSocketFile()
     val args = Array(
       "bsp",
+      "--verbose",
```

Comment:
This produce quite a lot of noise iirc, are you sure it should be enabled by default?